chart/values.yaml

```diff
@@ -53,7 +53,7 @@ controller:
     host: "" # host to use to serve requests
     port: 9443 # port to use to serve requests
   safeMode:
-    enable: false
+    enable: true
```

Comment:
Is it expected to enable safeguards by default? It has potentially high impact on most people using the open source chart.